doc/how-tos/rotate-key/nodejs/README.md

```diff
@@ -0,0 +1,76 @@
+# Rotate a Key
+
+Indy-SDK Developer Walkthrough #2, NodEJS Edition
+
+[ [Python](../python/README.md) | [Java](../java/README.md) | [.NET](../../not-yet-written.md) | [Node.js](../../not-yet-written.md) | [Objective C](../../not-yet-written.md) | [Rust](../rust/README.md)]
+
```

Comment:
Please remove `[Node.js](../../not-yet-written.md) |`

Reply:
@Artemkaaas Done :-)